index.d.ts

```diff
@@ -0,0 +1,9 @@
+type FastifyConstraintsOptions = {
+  constraints: { [name: string]: any}
```

Comment:
formatting (missing space before closing brace)

Reply:
I'll be removing the file as the new implementation won't take any options